spec/spans.yml

```diff
@@ -0,0 +1,174 @@
+groups:
+  - id: span.graphql.server.operation
```

Comment:
To clarify, this span representing the complete GraphQL request processing, correct?
If so, wouldn't it be more accurate to have its id be `span.graphql.server.request`?